committeeoversightapp/models.py

```diff
@@ -9,17 +19,88 @@ class HearingCategoryType(models.Model):
+
 
 class WitnessDetails(models.Model):
     witness = models.ForeignKey(EventParticipant, on_delete=models.CASCADE)
-    document = models.ForeignKey(EventDocument, null=True, blank=True, on_delete=models.CASCADE)
-    organization = models.CharField(max_length=100, null=True, blank=True, primary_key=False)
+    document = models.ForeignKey(EventDocument,
+                                 null=True,
+                                 blank=True,
+                                 on_delete=models.CASCADE)
+    organization = models.CharField(max_length=100,
+                                    null=True,
+                                    blank=True,
+                                    primary_key=False)
     retired = models.BooleanField(default=False)
 
+
 class Committee(models.Model):
     lugar_id = models.IntegerField(null=True, blank=True, primary_key=False)
-    lugar_name = models.CharField(max_length=200, null=True, blank=True, primary_key=False)
-    organization = models.ForeignKey(Organization, null=True, blank=True, on_delete=models.CASCADE)
+    lugar_name = models.CharField(max_length=200,
+                                  null=True,
+                                  blank=True,
+                                  primary_key=False)
+    organization = models.ForeignKey(Organization,
+                                     null=True,
+                                     blank=True,
+                                     on_delete=models.CASCADE)
+
+
+class StaticPage(Page):
+    body = StreamField([
+        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
+        ('paragraph', blocks.RichTextBlock()),
+        ('image', ImageChooserBlock()),
+        ('button',
+         blocks.StructBlock([
+            ('button_text', blocks.CharBlock()),
+            ('button_link', blocks.URLBlock())
+            ],
+         icon='site'))
+    ])
+
+    # Editor configuration
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('body'),
+    ]
+
+
+class CategoryDetailPage(Page):
+    # model page method adapted from https://timonweb.com/tutorials/how-to-hide-and-auto-populate-title-field-of-a-page-in-wagtail-cms/
+    category = models.ForeignKey(
+        HearingCategoryType,
+        blank=False,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text="Select a category for this page."
+    )
+
+    body = StreamField([
+        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
+        ('paragraph', blocks.RichTextBlock()),
+        ('image', ImageChooserBlock()),
+        ('button',
+         blocks.StructBlock([
+            ('button_text', blocks.CharBlock()),
+            ('button_link', blocks.URLBlock())
+            ],
+         icon='site'))
+    ])
+
+    # Editor configuration
+    content_panels = [
+        FieldPanel('category'),
+        StreamFieldPanel('body'),
+    ]
+
+    def save(self, *args, **kwargs):
```

Comment:
Overriding the default `Page` `save()` function here to make sure the page title and draft title match the category selected when the page is saved